server/routes/bookingJourney/selectAdditionalSupportController.ts

```diff
@@ -0,0 +1,49 @@
+import type { RequestHandler } from 'express'
+import { ValidationChain, body, validationResult } from 'express-validator'
+import { BookerService } from '../../services'
+
+export default class AdditionalSupportController {
```

Comment:
Class name doesn't match filename as other existing controllers do. Not too fussed about naming convention but we should be consistent (we may want to tweak names of all these controllers once we hear back from Ed on what URLs/page names should be)